src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -2,29 +2,27 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
-
 import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
 import javax.validation.constraints.NotNull;
-import javax.persistence.Column;
 
-@AllArgsConstructor
-@Setter
 @Getter
+@Setter
 @Entity
+@NoArgsConstructor
+@AllArgsConstructor
 public class Product {
 
     @Id
     @NotNull
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
-    @Column
     @NotNull
     private String name;
     @NotNull
-    @Column
     private int price;
+    private int quantity;
+
```

Comment:
Uważam, że brakuje pola klasy OrderEntity z relacją @ManyToOne. To jest to pole, które ma być drugą stroną relacji. W 34 linii OrderEntity wspominałem, że nie ma na nie wskazanego parametru mappedBy.

Reply:
jest to rozwiązane przez @JoinColumn - więc tu chyba nie musi być tego pola o ile dobrze pamiętam.

Reply:
Dokładnie tak, jak pisze Mateusz. W encji OrderEntity jest @JoinColumn, gdzie zostanie dodana kolumna z listą obiektów typu Product. I to właśnie przy okazji doda się quantity.